src/processors/utils/approve.ts

```diff
@@ -25,16 +25,16 @@ export const approve = (
 };
 
 export const isApproved = async (
```

Comment:
Move entire method to EvmProviderAdapter. Why do we need a separate util for this?